Source/WebCore/platform/graphics/WOFFFileFormat.cpp

```diff
@@ -25,18 +25,22 @@
 
 #include "config.h"
 #include "WOFFFileFormat.h"
-#include <zlib.h>
 
 #include "SharedBuffer.h"
-#include <wtf/ByteOrder.h>
 
+#if !HAVE(WOFF_SUPPORT)
+#include <wtf/ByteOrder.h>
+#include <zlib.h>
```

Comment:
Please avoid using conditionals to guard headers unless it's required. Otherwise you're just increasing the odds of non-unified build failures or build failures in odd conditions. E.g. what if another file in the same unified source bundle requires ByteOrder.h but forgets to include it?

Reply:
I agree with you just a note though the implementation for everything is gated on `!HAVE(WOFF_SUPPORT)` so in this case its perfectly fine.

Reply:
What I mean is: since the build won't fail if you remove the guard, remove it. Try to keep source files containing the same header files regardless of build options to the greatest extent possible. Minimizing includes is valuable but not at the cost of introducing more variability in what gets included in different configs.